projects/demo/src/pages/kit/number/number-mask-doc.template.html

```diff
@@ -102,10 +102,39 @@
             <number-mask-doc-example-4></number-mask-doc-example-4>
         </tui-doc-example>
 
+        <tui-doc-example
+            id="minus-sign"
+            heading="Minus sign"
+            [content]="customMinusSignExample5"
+            [description]="customMinusSignDescription"
+        >
+            <ng-template #customMinusSignDescription>
+                <p>
+                    Use
+                    <code>minusSign</code>
+                    parameter to configure the character which indicates that a number is negative.
+                </p>
+                <p>
+                    In this example
+                    <a
+                        href="https://symbl.cc/en/2010"
+                        rel="noreferrer"
+                        target="_blank"
+                        tuiLink
```

Comment:
Add `TuiLinkModule` import

Otherwise:
<img width="1014" alt="Снимок экрана 2024-03-06 в 14 19 24" src="https://github.com/taiga-family/maskito/assets/35179038/f4ff1fc5-7caa-457d-8f99-13146b1b84da">